utplsql-maven-plugin/src/test/resources/type-mapping-project/pom.xml

```diff
@@ -0,0 +1,94 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.utplsql</groupId>
+    <artifactId>utplsql-maven-plugin-test</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>maven-plugin</packaging>
+
+    <name>utplsql-maven-plugin Maven Plugin Test</name>
+    
+    
+     <properties>
+      	<dbUrl>jdbc:oracle:thin:@127.0.0.1:1521:xe</dbUrl>  
+		<!-- 
+ 		<dbUser>ut3</dbUser> 
+  		<dbPass>XNtxj8eEgA6X6b6f</dbPass> 
+  		-->
+    </properties>
+    
+    <build>
+    
+        <plugins>
+            <plugin>
+                <groupId>${pom.groupId}</groupId>
+                <artifactId>utplsql-maven-plugin</artifactId>
+                <version>${pom.version}</version>
+                <goals>
+                    <goal>test</goal>
+                </goals>
+                <configuration>
+                	<!--  Mandatory Attributes -->
+                	
+					<ignoreFailure>false</ignoreFailure>
+					
+					<paths>
```

Comment:
Could we change from `paths` to `suitepaths`? `paths` seem too ambiguous and not self-descriptive enough.

Reply:
Followed in #17